messages/password.generate.json

```diff
@@ -1,15 +1,18 @@
 {
-  "description": "generate a password for scratch org users\nGenerates and sets a random password for one or more scratch org users. Targets the usernames listed with the --onbehalfof parameter or the --targetusername parameter. Defaults to the defaultusername.\n\nIf you haven’t set a default Dev Hub, or if your scratch org isn’t associated with your default Dev Hub, --targetdevhubusername is required.\n\nTo see a password that was previously generated, run \"sfdx force:user:display\".",
+  "description": "generate a password for scratch org users\nGenerates and sets a random password for one or more scratch org users. Targets the usernames listed with the --onbehalfof parameter or the --targetusername parameter. Defaults to the defaultusername.\n\nIf you haven’t set a default Dev Hub, or if your scratch org isn’t associated with your default Dev Hub, --targetdevhubusername is require.\n\nTo change the password strength, set the --complexity parameter to a value between 0 and 5. \nEach value specifies the types of characters used in the generated password: \n\n0 - lower case letters only.\n1 - lower case letters and numbers only.\n2 - lower case letters and symbols only.\n3 - lower and upper case letters and numbers only\n4 - lower and upper case letters and symbols only\n5 - lower and upper case letters and numbers and symbols only \n\nTo see a password that was previously generated, run \"sfdx force:user:display\".",
```

Comment:
```suggestion
  "description": "generate a password for scratch org users\nGenerates and sets a random password for one or more scratch org users. Targets the usernames listed with the --onbehalfof parameter or the --targetusername parameter. Defaults to the defaultusername.\n\nIf you haven’t set a default Dev Hub, or if your scratch org isn’t associated with your default Dev Hub, --targetdevhubusername is required.\n\nTo change the password strength, set the --complexity parameter to a value between 0 and 5. \nEach value specifies the types of characters used in the generated password: \n\n0 - lower case letters only.\n1 - lower case letters and numbers only.\n2 - lower case letters and symbols only.\n3 - lower and upper case letters and numbers only\n4 - lower and upper case letters and symbols only\n5 - lower and upper case letters and numbers and symbols only \n\nTo see a password that was previously generated, run \"sfdx force:user:display\".",
```

Reply:
Ooof, this is a long description :).  I'm trusting that it looks okay in the --help output.  I'll take a look when it's in latest-rc, and maybe rework it if it looks funky.